### DISCUSSION TITLE
PrimeNG PanelMenu navigation

### DISCUSSION THREAD

**Discussion by aydinburak** [primefaces/community#1564] [PrimeNG]
Hello,

I've been working with PrimeNG for a bit over a month now. I haven't had any problem with any components so far but now I've been struggling with PanelMenu navigation for the last 2 days. I created the json for menu items and set the panel menu in html. And now I can't seem to make navigation work. Any source I've found online does same things but for some reason any of them works for me.

![image](https://github.com/primefaces/community/assets/7476606/33d79cbb-ac93-4d12-a941-d150f0e30cb4)

I've tried routerLink, url and command=>navigate separately and none of them worked.

![image](https://github.com/primefaces/community/assets/7476606/fbee69f2-5be6-4b55-815a-bb112d4063f6)

I can provide more information upon asking

Any help would be appreciated!

Edit: Trying couple of things made me realize sub-menu items don't work with navigation. Parents work just fine.

Edit2: After seeing what the final html looks like, I realized router-link is added to 'a' tag and not 'li' tag. I edited the html to leave just 'a' tag and it worked. I don't know how to solve this exactly but working on it. 

![image](https://github.com/primefaces/community/assets/7476606/6f4590a5-ed44-4064-bb23-ed36328ca270)

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue?
